chat-plugins/info.js

```diff
@@ -586,6 +586,7 @@ exports.commands = {
 				dex[pokemon] = template;
 			}
 		}
+		dex = JSON.parse(JSON.stringify(dex)); // Don't modify the original template (when compiling learnsets)
```

Comment:
That's about 9x slower than using JSON:
```
JSON clone: 130ms
JSON clone: 62ms
JSON clone: 78ms
JSON clone: 77ms
sugar clone: 731ms
[slow] 762ms - kota <127.0.0.1>: |/ds sd
sugar clone: 679ms
[slow] 718ms - kota <127.0.0.1>: |/ds sd
sugar clone: 660ms
[slow] 689ms - kota <127.0.0.1>: |/ds sd
sugar clone: 616ms
[slow] 642ms - kota <127.0.0.1>: |/ds sd
sugar clone: 614ms
```

Maybe push dexsearch into its own file and only compile the learnsets once?